projects/app/src/app/(sidenav)/learn/index.tsx

```
@@ -18,13 +20,34 @@ export default function IndexPage() {
-      for (const [, { skill }] of ratingHistory) {
-        if (
-          `hanziWord` in skill &&
-          !recentHanziWords.includes(skill.hanziWord)
-        ) {
-          recentHanziWords.push(skill.hanziWord);
-          if (recentHanziWords.length === 10) {
+      pushLoop: for (let [, { skill }] of ratingHistory) {
+        switch (skillType(skill)) {
+          case SkillType.EnglishToHanziWord:
+          case SkillType.HanziWordToEnglish:
+          case SkillType.HanziWordToEnglish:
```

Biome (1.9.4)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)